cif_core.dic

```diff
@@ -2014,81 +2014,97 @@ save_diffrn_refln.counts_bg_1
+
+    Standard uncertainties should not be quoted for these values.
+    If the standard uncertainties differ from the square root of
+    the number of counts, _diffrn_refln.intensity_* should be used.
 ;
     _name.category_id             diffrn_refln
-    _name.object_id               counts_bg_1_su
-    _name.linked_item_id          '_diffrn_refln.counts_bg_1'
-    _units.code                   none
-
-    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+    _name.object_id               counts_bg_1
+    _type.purpose                 Number
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Integer
+    _enumeration.range            0:
+    _units.code                   counts
 
 save_
 
 save_diffrn_refln.counts_bg_2
 
     _definition.id                '_diffrn_refln.counts_bg_2'
     _alias.definition_id          '_diffrn_refln_counts_bg_2'
-    _name.category_id             diffrn_refln
-    _name.object_id               counts_bg_2
-
-    _import.get                   [{'file':templ_attr.cif  'save':diffr_counts}]
-
-save_
-
-save_diffrn_refln.counts_bg_2_su
-
-    _definition.id                '_diffrn_refln.counts_bg_2_su'
-    _definition.update            2021-09-23
+    _definition.update            2022-11-22
     _description.text
 ;
-    Standard uncertainty of _diffrn_refln.counts_bg_2.
+    The measured background counts on the other side of the
+    peak to the measurement of _diffrn_refln.counts_bg_1
+    when measuring using a point detector.
+
+    Note that counts-per-second values should be converted to
+    total counts.
+
+    Standard uncertainties should not be quoted for these values.
+    If the standard uncertainties differ from the square root of
+    the number of counts, _diffrn_refln.intensity_* should be used.
 ;
     _name.category_id             diffrn_refln
-    _name.object_id               counts_bg_2_su
-    _name.linked_item_id          '_diffrn_refln.counts_bg_2'
-    _units.code                   none
-
-    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+    _name.object_id               counts_bg_2
+    _type.purpose                 Number
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Integer
+    _enumeration.range            0:
+    _units.code                   counts
 
 save_
 
 save_diffrn_refln.counts_net
 
     _definition.id                '_diffrn_refln.counts_net'
     _alias.definition_id          '_diffrn_refln_counts_net'
+    _definition.update            2022-11-29
+    _description.text
+;
+    Counts measured in the reflection peak after background
+    subtraction.
+
+    Note that counts-per-second values should be converted to
+    total counts.
+
+    The value for this data name must be derived from count values.
+    If not, _diffrn_refln.intensity_* should be used.
+;
```

Comment:
If I measure counts in the peak P for 3 times longer than the background B, then I could calculate `counts_net` as P-3B, or I could calculate as P/3 - B. Both are "derived from count values" and both are correct.  If we want to use this for area detector single-crystal data, then our final value could be more complicated, but still only involve counts and relative scaling factors.

For avoidance of the above ambiguity I think we need to be more specific and state that background counts should be scaled to the peak counts before subtraction. This is the usual practice so I don't think we are changing anything here.

Reply:
I hadn't thought that someone _wouldn't_ scale bkg to peak first! I'll add that.